Dockerfile.template

```diff
@@ -2,6 +2,10 @@ FROM alpine:{{ .alpine }}
 
 RUN apk add --no-cache \
 		ca-certificates \
+{{ if ["19.03", "19.03-rc", "20.10-rc"] | index(env.version) | not then ( -}}
```

Comment:
Very minor nit that this should probably be `env.version == "20.10"`, but it's harmless and the same either way, and hopefully temporary :smile:

Reply:
oh! wrote this based on the existing example I found. didn't try other options, but yes, makes sense.